### DISCUSSION TITLE
Notification bout onedrive Clien obsolete

### DISCUSSION THREAD

**Discussion by joseluisgs** [abraunegg/onedrive#1982] [Q&A] [answered]
Hi
I have thos notification in Ubuntu 22.04
```
WARNING: Your onedrive client version is obsolete and unsupported. Please upgrade your client version. 
```
My version is onedrive v2.4.17-1~np3

Could you help me?
Thanks

**Comment by abraunegg** [comment, marked as answer]
@joseluisgs 
You need to upgrade your client. You will be able to upgrade your client when the [OpenSuSE Build Service](https://build.opensuse.org/package/show/home:npreining:debian-ubuntu-onedrive/onedrive) repository is updated.

**Edit:** Which is now updated ...

**Reply by joseluisgs** [reply]
Hi, thanks again. I have updated this morning!
Thanks again for your client, it is awasome!
:)

**Reply by landall** [reply]
> I cannot update it in CentOS with harbottle repo.



> @joseluisgs You need to upgrade your client. You will be able to upgrade your client when the [OpenSuSE Build Service](https://build.opensuse.org/package/show/home:npreining:debian-ubuntu-onedrive/onedrive) repository is updated.
> 
> **Edit:** Which is now updated ...

I cannot update it in CentOS with epel repo.

**Reply by rrodrigueznt** [reply]
@landall 

> > I cannot update it in CentOS with harbottle repo.

If you are planning a strategy to maintain OneDrive update on CentOS, I think it is worth a look to the [RPM Package Build Process](RPM Package Build Process) doc and to @abraunegg comments on this [discussion thread](https://github.com/abraunegg/onedrive/discussions/2275).

OneDrive 2,4,22 is working here like a charm in a CentOS Stream 9 testing virtual box running on **Windows Subsystem for Linux (WSL)**

**Reply by landall** [reply]
> @landall
> 
> > > I cannot update it in CentOS with harbottle repo.
> 
> If you are planning a strategy to maintain OneDrive update on CentOS, I think it is worth a look to the [RPM Package Build Process](RPM Package Build Process) doc and to @abraunegg comments on this [discussion thread](https://github.com/abraunegg/onedrive/discussions/2275).
> 
> OneDrive 2,4,22 is working here like a charm in a CentOS Stream 9 teting virtual box running on **Windows Subsystem for Linux (WSL)**

Let me try to contact the maintainer of harbottle repo. 
It is strange that this app is marked with a EPEL8 tag but not in EPEL8 repo , in https://koji.fedoraproject.org/koji/packageinfo?packageID=26044

**Comment by abraunegg** [comment]
@landall 

> I cannot update it in CentOS with epel repo.

I have zero control over any packages or distributions who package the client. The EPEL packages come from Fedora. Suggest that you complain to the package maintainer (Zamir Sun) to update the package and request that the package is updated in a more timely manner.

There is *zero* I can do to assist - you need to bug the package maintainer.

Refer to:
* https://koji.fedoraproject.org/koji/packageinfo?packageID=26044
* https://bugzilla.redhat.com/buglist.cgi?quicksearch=onedrive

As an alternative, you can, as @rrodrigueznt has suggested above - build your own package - it is a very painless process and works very well.
